### ISSUE TITLE
Bug in draw_callback_postview self.handle_exception()

### ISSUE THREAD

**Issue by carter2422** [CGCookie/retopoflow#310]
I've never been able to narrow down when or why this happens, only that when it happens I get the below error spit to the console with each draw update. It doesn't seem to break anything, and doesn't result in a crash, but it'd be nice to track it down. 

It's only in Polystrips I think.

```
Traceback (most recent call last):
  File "/Users/jonathanwilliamson/Dropbox (Personal)/blenderDev/addons/cgCookie/scripts/addons/retopoFlow/modaloperator.py", line 164, in draw_callback_postview
    self.handle_exception()
  File "/Users/jonathanwilliamson/blender-dev/blender-master/build_darwin/bin/blender.app/Contents/MacOS/./../Resources/2.78/scripts/modules/bpy_types.py", line 601, in __getattribute__
    properties = StructRNA.path_resolve(self, "properties")
ReferenceError: StructRNA of type CGC_Polystrips has been removed
Traceback (most recent call last):
  File "/Users/jonathanwilliamson/Dropbox (Personal)/blenderDev/addons/cgCookie/scripts/addons/retopoFlow/modaloperator.py", line 170, in draw_callback_postpixel
    self.draw_postpixel(context)
  File "/Users/jonathanwilliamson/blender-dev/blender-master/build_darwin/bin/blender.app/Contents/MacOS/./../Resources/2.78/scripts/modules/bpy_types.py", line 601, in __getattribute__
    properties = StructRNA.path_resolve(self, "properties")
ReferenceError: StructRNA of type CGC_Polystrips has been removed
```

**Comment by vxlcoder** [comment]
@carter2422 was there anything else reported to console? It looks as though the actual problem occurred just prior to this trace.

**Comment by carter2422** [comment]
I'll try and find more info.

**Comment by vxlcoder** [comment]
I believe this issue applies to v1.x only, so I'm going to close this issue now that v2.0 is officially released (see `release/2.0.0` branch or Blender Market page).  If this is still an issue with v2.0, please reopen, and post more details.  Thanks for reporting this issue, and we look forward to seeing how v2.0 helps your retopo workflow!
